src/main/kotlin/sidechain/eth/util/Crypto.kt

```diff
@@ -12,13 +11,13 @@ import java.math.BigInteger
  * @param toSign data to sign
  * @return signed data
  */
-fun signUserData(toSign: String): String {
+fun signUserData(ethereumConfig: EthereumConfig, toSign: String): String {
     // TODO luckychess 26.06.2018 D3-100 find a way to produce correct signatures locally
-    val deployHelper = DeployHelper()
-    val parity = Parity.build(HttpService(CONFIG[ConfigKeys.testEthConnectionUrl]))
+    val deployHelper = DeployHelper(ethereumConfig)
+    val parity = Parity.build(HttpService(ethereumConfig.url))
     parity.personalUnlockAccount(
         deployHelper.credentials.address,
-        CONFIG[ConfigKeys.relayRegistartionEthCredentialPassword]
+        ethereumConfig.credentialsPassword
```

Comment:
May be it would be better to load address from config as well and get rid of ```DeployHelper``` in this function?

Reply:
I think it is internal representation of Eth, and it is better to use provided interface.